saleor/graphql/plugins/mutations.py

```diff
@@ -99,7 +99,9 @@ def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
         plugin_id = cleaned_data["plugin"].PLUGIN_ID
         channel_slug = cleaned_data["channel_slug"]
         input_data = cleaned_data["data"]
+
         manager = get_plugin_manager_promise(info.context).get()
         manager.save_plugin_configuration(plugin_id, channel_slug, input_data)
-        manager = get_plugins_manager()
+
+        manager = get_plugins_manager(allow_replica=info.context.allow_replica)
```

Comment:
I'm wondering if here we shouldn't explicitly pass `False` but maybe let's see what others think.

Reply:
In such case, should we also force `allow_replica=False` [here](https://github.com/saleor/saleor/pull/14340/files/204cab3c7a82ace09dc0bab2261e7b0524db3788#diff-8c0b159a7dafb59bbe23bc4b82fb8e294a547e5171900b41bd3c696386d474afR103) too?